examples/geojson/ai-ecosystem-example.json

```
@@ -0,0 +1,1115 @@
+{
+    "fiboa": {
+        "fiboa_version": "0.2.0",
+        "fiboa_extensions": [
+          "https://fiboa.github.io/ai-ecosystem-extension/v0.1.0/schema.yaml"
+        ]
+    },
+    "type": "FeatureCollection",
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {
+                "area": 0.011851919813431496,
+                "perimeter": 49.937117861428113,
+                "determination_method": "administrative",
+                "determination_datetime": "2022-11-30",
```

```suggestion
                "determination_datetime": "2022-11-30",
```

This must always be a date and a time to be valid, e.g. `2022-11-30Z00:00:00Z`

---

Apologies, will change that.

---

Thanks.

If that's changed I hope the CI succeeds, if not we also need to fix whatever the CI complains about :-)